main.cpp

```diff
@@ -209,31 +209,31 @@ int main()
     soundOsc(1,0);
 
     //2)
-    auto sumChannels = sumLR(0,0);
+    auto sumChannels = sumLR(0, 0);
 
     //3)
     auto wave = waveType(0);
 
     //4)
-    auto envelopeOut = amplitudeEnv(0,0);
+    auto envelopeOut = amplitudeEnv(0, 0);
 
     //5)
-    auto userSelection = selectParam(2,0);
+    auto userSelection = selectParam(2, 0);
     
     //6)
-    auto userDisplay = displaySelectedOscType(0,0);
+    auto userDisplay = displaySelectedOscType(0, 0);
 
     //7)
-    auto connectionStatus = isSomethingConnected(0,0);
+    auto connectionStatus = isSomethingConnected(0, 0);
 
     //8)
-    auto batteryInfo = checkBatteryStatus(0,0,1);
+    auto batteryInfo = checkBatteryStatus(0, 0, 1);
     
     //9)
-    auto inputPreAmp = detectInputType(0,0);
+    auto inputPreAmp = detectInputType(0, 0);
     
     //10)
-    auto arp = activateArp(0,0);
+    auto arp = activateArp(0, 0);
```

Comment:
A lot of these functions want a boolean value for one of the parameters.   You are passing integers.
you need to pass `true` or `false` wherever a boolean is expected
- [ ] revise